test/src/test/java/hudson/util/AtomicFileWriterPerfTest.java

```diff
@@ -23,6 +24,7 @@
      * So using slightly more than the worse value obtained above should avoid making this flaky and still catch
      * <strong>really</strong> bad performance regressions.
      */
+    @Ignore("TODO often fails in CI")
```

Comment:
https://github.com/jenkinsci/jenkins/pull/3797#pullrequestreview-183418725

Reply:
As above, perhaps just assume not running in CI?

Reply:
I like that idea, because this test is unlikely to ever fail in a normal environment (I mean something built after ~1995), and at the same time I feel a bit bad if tests run locally are not the same as in CI. I see potential interesting headaches for some people if some tests fail locally but not in CI (though I suppose a comment would help about this...)

Reply:
> this test is unlikely to ever fail in a normal environment

Possibly so, but performance tests are not suitable as merge/deploy gates—I could be patching `README.md` and the build might be marked as a failure for reasons unrelated to my change. Or, as in #3778, I could be trying to get a binary published for some unrelated fix and be stymied by this flake.

Better to have these in a separate job (repo?) where they are run say on a daily basis with some kind of tooling to track trends and alert developers to consistent regressions.